File: dataset/python2java/python/SUM_PAIRWISE_PRODUCTS_1.py
def f_gold ( n ) :
    multiTerms = n * ( n + 1 ) // 2
    sm = multiTerms
    for i in range ( 2 , n + 1 ) :
        multiTerms = multiTerms - ( i - 1 )
        sm = sm + multiTerms * i
    return sm

File: result/translate/transcoder/python2java/java/SUM_PAIRWISE_PRODUCTS_1.java
static double f_gold ( int n ) {
  int multiTerms = n * ( n + 1 ) / 2 ;
  double sm = multiTerms ;
  for ( int i = 2 ;
  i <= n ;
  i ++ ) {
    multiTerms = multiTerms - ( i - 1 ) ;
    sm = sm + multiTerms * i ;
  }
  return sm ;
}
